components/application-operator/charts/application/values.yaml

```diff
@@ -29,7 +29,7 @@ eventService:
       pullPolicy: IfNotPresent
     args:
       externalAPIPort: &eventServiceExternalAPIPort 8081
-      eventsTargetURL: http://core-publish.kyma-system.svc.cluster.local:8080/v1/events
+      eventsTargetURL: http://event-bus-publish.kyma-system.svc.cluster.local:8080/v1/events
```

Comment:
Does this change requires releasing the operator first?

Reply:
Yes! in a following PR